src/main/java/wromaciej/hvac_sim/thermo/devices/model/basic/Computable.java

```diff
@@ -0,0 +1,6 @@
+package wromaciej.hvac_sim.thermo.devices.model.basic;
+
+public interface Computable {
+
+    void compute();
```

Comment:
what will that compute and why it does not return computation result?

Reply:
its just a sketch now, but i changed void to boolean to check the result

Reply:
why boolean? I thought you should rather return some numeric value. however i don;t know what that computation is intended to be.